app/controllers/hometasks_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class HometasksController < ApplicationController
+  def index
+    if current_user.student?
+      @hometasks = []
+      current_user.group.courses.each do |course|
+        @hometasks << Hometask.for_course(course).last
+      end
```

Comment:
Move lines 6-9 in separate method student_hometasks or move to User model - create method hometasks.